.github/workflows/node.js.yml

```diff
@@ -26,10 +26,8 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
+          cache: 'npm'
       - run: npm ci
-      - run: git submodule update --init --recursive
-      - run: cd ./submodules/universalprofile-smart-contracts && npm ci && cd ./submodules/ERC725/implementations && npm ci
-      - run: cd ./submodules/universalprofile-smart-contracts && npx truffle build
       - run: npm run build --if-present
```

Comment:
No lint check? I also see no 'lint' script in the package.

This action is cool, the output is better than just running 'npm run lint'

```yaml
- name: Lint
        uses: wearerequired/lint-action@v1
        with:
          eslint: true
          eslint_extensions: ts
```

Reply:
There is also this super_linter github action. That combine multiple linters in one. If useful.
https://github.com/github/super-linter